packages/twenty-front/src/modules/object-record/components/RecordShowPage.tsx

```diff
@@ -260,7 +260,7 @@ export const RecordShowPage = () => {
                           key={record.id + fieldMetadataItem.id}
                           value={{
                             entityId: record.id,
-                            maxWidth: 272,
+                            maxWidth: 200,
```

Comment:
should be 200 if label is displayed, 272 if label is not displayed

Reply:
Hi Charles ,
I don't this is right way to way that but i tried.  just let me know where i am wrong .
Thank you .